src/test/java/org/jabref/logic/shared/DBMSProcessorTest.java

```diff
@@ -311,6 +364,24 @@ private static BibEntry getBibEntryExample() {
                 .withCiteKey("nanoproc1994");
     }
 
+    private static BibEntry getBibEntryExample2() {
```

Comment:
Please give the method a more meaningful name. For instance `getShelahBibEntry`.